aspnetcore/razor-pages/ui-class.md

```diff
@@ -446,6 +446,33 @@ In the sample download, rename *WebApp1/Areas/MyFeature2* to *WebApp1/Areas/MyFe
 
 Copy the *RazorUIClassLib/Areas/MyFeature/Pages/Shared/_Message.cshtml* partial view to *WebApp1/Areas/MyFeature/Pages/Shared/_Message.cshtml*. Update the markup to indicate the new location. Build and run the app to verify the app's version of the partial is being used.
 
+One must enable `Razor Pages` in the web app to make sure the page based routing works. For example, in an MVC application, register the `Razor Pages` services and map the pages with endpoint routing,
+
+```csharp
```

Comment:
Finish moving this to file. Use snippet syntax to highlight the two lies
Example syntax
`[!code-csharp[](start-mvc/sample/src/MvcMovie/Models/MovieDateFormat.cs?name=snippet1&highlight=2,10)]`